plugin/src/main/scala/com/softwaremill/clippy/AdviceLoader.scala

```diff
@@ -15,7 +15,8 @@ class AdviceLoader(global: Global, url: String, localStoreDir: File)(implicit ec
   private val OneDayMillis = 1000L * 60 * 60 * 24
 
   private val localStore = new File(localStoreDir, "clippy.json.gz")
-  private val projectAdviceFile = new File(".clippy.json")
+  private val projectRoot = sys.props.get("user.dir").getOrElse(".")
```

Comment:
@kciesielski You were right that I am using user.dir incorrectly.  I am preparing a new pr now that has a better method of locating the project root directory.